frontend/src/components/MapView/DateSelector/TimelineItems/index.tsx

```diff
@@ -136,7 +127,7 @@ const TimelineItems = memo(
         dateRange[0].value,
       ).toDateString();
 
-      const hello = [intersectionDateItems, ...selectedLayerDateItems].map(
+      const layers = [intersectionDateItems, ...selectedLayerDateItems].map(
```

Comment:
You can return it directly don't need the constant here its redundant

Reply:
Done